.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+on:
+  workflow_dispatch:
```

Comment:
Releases can be triggered with the "Run workflow" button on the GitHub UI (if you want we can also configure to automatically release once something is merged into `main`)

<img width="1158" alt="Screenshot 2024-10-07 at 09 58 06" src="https://github.com/user-attachments/assets/4669c23c-41eb-4b66-9128-602e69cd9a3d">